_search-plugins/search-pipelines/score-ranker-processor.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Score ranker
+has_children: false
+parent: Search processors
+grand_parent: Search pipelines
+nav_order: 117
+---
+
+# Score ranker processor
+Introduced 2.19
+{: .label .label-purple }
+
+The `score-ranker-processor` is a rank-based search phase results processor that runs between the query and fetch phases of search execution. It intercepts the query phase results and then uses the reciprocal rank fusion (RRF) algorithm to combine different query clauses to produce the final ranked list of search results. RRF is a method for combining multiple queries by scoring each document based on the reciprocal of its rank for each query and then adding these scores to create a final, unified ranking.
```

Comment:
"results processor" should technically be "result processor". Is that acceptable, or can we write around it?

Reply:
The parameter for these processors is named `phase_results_processors` so I think it's best to keep "results".